webpack.common.js

```diff
@@ -0,0 +1,81 @@
+const path = require("path");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const CopyPlugin = require("copy-webpack-plugin")
+const BrowserSyncPlugin = require('browser-sync-webpack-plugin')
+const PACKAGE = require('./package.json')
```

Comment:
Waarom in hoofdletters? en `theme` dan niet?

Reply:
hmm, dan zou ik het `PackageJson` noemen fzo, want anders gaat dat maybe issue geven als er iemand zo op case insensitive stuff zit.